components/org.wso2.carbon.identity.captcha/src/main/java/org/wso2/carbon/identity/captcha/util/CaptchaUtil.java

```diff
@@ -418,6 +437,20 @@ private static void setReCaptchaConfigs(Properties properties) {
         }
         CaptchaDataHolder.getInstance().setReCaptchaRequestWrapUrls(reCaptchaRequestWrapUrls);
 
+        String reCaptchaScoreThreshold = properties.getProperty(CaptchaConstants.RE_CAPTCHA_SCORE_THRESHOLD);
```

Comment:
can we introduce a separate private method to get the threshold value. if the threshold is empty, return the  static value